src/lib/UI/Config/Provider/Languages.php

```diff
@@ -27,8 +27,8 @@ class Languages implements ProviderInterface
     /** @var string[] */
     private $siteAccesses;
 
-    /** @var \eZ\Publish\Core\MVC\Symfony\SiteAccess */
-    private $siteAccess;
+    /** @var \eZ\Publish\Core\MVC\Symfony\SiteAccess\SiteAccessService */
+    private $siteAccessService;
```

Comment:
You shouldn't rely on concrete implementation. Use corresponding interface instead.